### PR TITLE
Added TestFairy code samples

### PR DESCRIPTION
**Pull request by diegoperini**

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -240,13 +240,13 @@ PODS:
   - React-jsinspector (0.63.4)
   - react-native-biometrics (2.1.4):
     - React
-  - react-native-camera (3.40.0):
```
Comment (gmegidish) [inline ios/Podfile.lock:243]: @diegoperini do we need this file changed? can we have it exactly as it was originally?

File: src/js/App.js [hunk 2]
```diff
@@ -37,6 +38,34 @@ QuickActions.setShortcutItems([
...
+		//   - custom attributes set in the code w/ TestFairy.setAttribute()
+		//   - events sent in the code w/ TestFairy.addEvent()
+		//
+		// and many others.
+		//
+		// All of this data will be available in your TestFairy web dashboard as well as the REST API.
+		//
+		////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+		// TestFairy.setServerEndpoint("https://your.privatecloud.example.com") // Private cloud only
+		TestFairy.begin('SDK-gLeZiE9i');
```
Comment (gmegidish) [inline src/js/App.js]: Please have this file read from assets/user_data.json, see dmaf for reference.

File: src/js/screens/CheckoutScreenOne.js [hunk 3]
```diff
@@ -14,6 +14,7 @@ import SecondaryHeader from '../components/SecondaryHeader';
 import { ShoppingCart } from '../shopping-cart';
 import { SCREENS } from '../config/Constants';
 import { handleQuickActionsNavigation } from '../config/QuickActionsNavigation';
+import TestFairy from 'react-native-testfairy';
```
Comment (gmegidish) [inline src/js/screens/CheckoutScreenOne.js:17]: Coding convension

File: ios/Podfile.lock (hunk 1 above)
Comment (diegoperini) [reply ios/Podfile.lock:243]: We need it changed at least for our package.

File: src/js/App.js (hunk 2 above)
Comment (diegoperini) [reply src/js/App.js]: Done

File: src/js/screens/CheckoutScreenOne.js (hunk 3 above)
Comment (diegoperini) [reply src/js/screens/CheckoutScreenOne.js:17]: What do you mean? Should this line be different or put somewhere else?

**Pull request merged**